m3u8/parser.py

```diff
@@ -320,9 +320,14 @@ def _parse_attribute_list(prefix, line, attribute_parser, default_parser=None):
 
     attributes = {}
     for param in params:
-        name, value = param.split("=", 1)
-        name = normalize_attribute(name)
+        param_parts = param.split("=", 1)
+        if len(param_parts) == 1:
```

Comment:
This change affects all tags - if there's some parameter without a `key=value` pair, it will get stuck into the empty string key.